openmrs-watcher/src/main/resources/application.properties

```diff
@@ -1,4 +1,7 @@
-camel.springboot.xml-routes=classpath*:camel/*.xml
+# Spring Boot Camel Configuration
+camel.springboot.name=OpenMRS EIP
+# Routes to include
+camel.springboot.routes-include-pattern=classpath:camel/*,classpath:camel-template/*,classpath:camel-rest/*
```

Comment:
I guess this will be xml files provided? In which case we'd have to worry about that end with the *.xml extension?